.github/workflows/nextjs.yml

```diff
@@ -6,9 +6,11 @@ on:
 
 jobs:
   check-next-website:
-    runs-on: ubuntu-22.04
```

Comment:
This actually made sense to have locked, since `ubuntu-latest` could be updated at any time and the pipeline break.

Reply:
Ok, we can revert it back. It was just incoherent that we had ubuntu:latest for testing the code and Ubuntu:22.x for deploying